docs/src/routes/docs/components/button.svx

```diff
@@ -133,6 +133,6 @@ The content of the Button. Icons, other components, everything is fair game here
 | **`$shadow-raised`** | The shadow of filled buttons on hover. | <ShadowPreview value={shadowRaised} /> |
 | **`$disabled`** | The text color of disabled buttons. | <ColorPreview value="#888888" /> |
 | **`$disabled-bg`** | The background color of filled disabled buttons. | <ColorPreview value="#D7D7D7" /> |
-| **`$bold-font-weight`** | The font weight of the buttons. | `500` |
+| **`$button-font-weight`** | The font weight of the buttons. | `$bold-font-weight` (`500`) |
```

Comment:
I don't think it's worth mentioning that `$button-font-weight` has a value of `$bold-font-weight`. Just `500` should do it

Reply:
I think it should just be `$bold-font-weight` since it is used as this variable's default. If users modify `$bold-font-weight` but not `$button-font-weight`, it will still change, to the confusion of those who see the default as `500`.

Reply:
In that case perhaps we should explicitly write out the value of the `$button-font-weight` in the `_variables.scss` instead. I don't like this referencing thing, it looks misleading as if changing one would change the other or something.